packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -56,7 +56,9 @@ export const ParticipantListWithHeading = (props: {
   const subheadingUniqueId = useId();
   const theme = useTheme();
   /* @conditional-compile-remove(total-participant-count) */
-  const totalParticipantCount = participantListProps.totalParticipantCount;
+  const totalParticipantCount = participantListProps.totalParticipantCount
+    ? participantListProps.totalParticipantCount
+    : 1;
```

Comment:
Why is `totalParticipantCount` undefined if you're the only one in the call? Shouldn't it correctly be 1?

Reply:
Another Q - In `paneTitleTrampoline` below, what happens on stable? Presumably the string is still `In this call {numberOfPeople}` so if we conditionally compile the `formatString` call won't we see in stable builds `In this call {numberOfPeople}`?

Reply:
it comes back from the sdk as undefined when you're alone for some reason, not totally sure why.

Reply:
great question, I believe when I originally did it it was not there as expected but I will take look

Reply:
oh right in stable we are still using the old control bar so it is not used at all 
![image](https://github.com/Azure/communication-ui-library/assets/94866715/e82eb851-ef4c-453c-81b2-05dfb8c80a0e)

Reply:
Oh this will be impacted by this active PR then! #3408

Reply:
Actually callwithchat stable already should use the people pane too

Reply:
> it comes back from the sdk as undefined when you're alone for some reason, not totally sure why.

Sounds like a bug :L

Reply:
![image](https://github.com/Azure/communication-ui-library/assets/94866715/f5ead171-b3cb-44ce-8fce-e9e2de675bed)
yeah seems like it

Reply:
> Actually callwithchat stable already should use the people pane too

good point will see what happens there

Reply:
yup was broken, I think I have a better fix coming